vscode/webviews/chat/cells/messageCell/human/editor/ToolboxButton.tsx

```diff
@@ -73,117 +71,132 @@ export const ToolboxButton: FC<ToolboxButtonProps> = memo(({ settings, api }) =>
+                        </h2>
+                        <div
+                            id="accordion-collapse-body"
+                            className="tw-ml-5 tw-p-5 tw-flex tw-flex-col tw-gap-5 tw-mt-1"
+                        >
+                            <div className="tw-text-sm">
+                                <span>
+                                    Agentic context can search your codebase, browse the web, execute
+                                    shell commands (when enabled), and utilize configured tools to
+                                    retrieve necessary context.
+                                    {/* TODO: Uncomment this when the docs is available */}
+                                    {/* <a href={AGENTIC_CONTEXT_DOCS}>Read the docs</a> to learn more. */}
+                                </span>
+                            </div>
+                            {/* Only shows the Terminal access option if client and instance supports it */}
+                            {settings.agent?.name && !settings.shell?.error && (
+                                <div>
+                                    <div
+                                        className="tw-flex tw-items-center tw-justify-between tw-w-full tw-font-medium tw-gap-3"
+                                        aria-label="terminal"
+                                    >
+                                        <span className="tw-flex tw-gap-2 tw-items-center">
+                                            <span className="tw-font-semibold tw-text-md">
+                                                Terminal access
+                                            </span>
+                                        </span>
+                                        <Switch
+                                            checked={settings.shell?.enabled}
+                                            disabled={isLoading || !!settings.shell?.error}
+                                            onChange={() =>
+                                                onSubmit({
+                                                    ...settings,
+                                                    shell: {
+                                                        enabled:
+                                                            !!settings.agent?.name &&
+                                                            !settings.shell?.enabled,
+                                                    },
+                                                })
+                                            }
+                                        />
+                                    </div>
+                                    <div className="tw-text-sm tw-mt-2">
+                                        Allows agents to execute terminal commands. When enabled, this
+                                        this tool can execute <code>ls</code>, <code>dir</code>,{' '}
```

Comment:
```suggestion
                                       tool can execute <code>ls</code>, <code>dir</code>,{' '}
```

Removing the extra _this_

Reply:
the copy was provided and approved by design so I'll leave it as is until we get a updated copy 👍